next-app/src/pages/myPage/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { MyPageView } from 'components/views/MyPage/MyPageView'
```

Comment:
```suggestion
import { MyPageView } from 'components/views/MyPage'
```
저는 views/MyPage 쪽 폴더 하위 구조를 이렇게 잡았으면 좋겠습니다
```
index.ts
MyPageContainer.tsx
```
```ts
// index.ts
export { default } from "./MyPageViewContainer"
```